src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResourcePagingTest.kt

```diff
@@ -224,4 +224,15 @@ class CourtResourcePagingTest : IntegrationTest() {
       .jsonPath("$.content[0].buildings[0].contacts[0].type").isEqualTo("TEL")
       .jsonPath("$.content[0].buildings[0].contacts[0].detail").isEqualTo("01545 570886")
   }
+
+  @Test
+  fun `a search finding multiple buildings and contacts should only return the court once`() {
+    // Preston Crown Court has 2 buildings each with 2 contacts
```

Comment:
is this comment related to the query?  There is only one `totalElements` returned anyway?

Reply:
The comment is to explain why I chose Preston Crown Court from the canned data set for this test.

Reply:
ah I understand now, since it is doing a `distinct` we were previously getting two records